credit.c

```diff
@@ -1,29 +1,83 @@
 #include <cs50.h>
 #include <stdio.h>
 
+// good job with formating. this is something that a lot of beginners mess up
+// and it shows that you're not a noob when you know how to format.
+// i personally like to put my open curly braces on the same line that i init my
+// function / if / while blocks but either works. if you work in an existing codebase,
+// follow the code formating rules they use.
+
+// ex: i do something like 
+// if (True == True) {
+//     printf("True");
+// }
+
+// in general try to separate your code in functions. it makes your code much more readible
+// and easier to debug
+
 int main(void)
 {
-    // Obtain user input for credit card number
-    long Number;
-    do
-    {
-        Number = get_long("Number: ");
-    }
-    while (Number < 0);
```

Comment:
i moved this code block after your declarations. there's a comment explaining below